esphome/components/esp32_ble_tracker/__init__.py

```diff
@@ -23,6 +23,8 @@
 CONF_SCAN_PARAMETERS = "scan_parameters"
 CONF_WINDOW = "window"
 CONF_ACTIVE = "active"
+CONF_CONTINUOUS = "continuous"
+CONF_ON_BLE_SCAN_END = "on_ble_scan_end"
```

Comment:
Probably dont need the `ble` here
```suggestion
CONF_ON_SCAN_END = "on_scan_end"
```

Reply:
I've implemented the suggested changes, checked locally and re-pushed.